Main

Fanis-Bazilisk opened the pull request.

Fanis-Bazilisk committed:
Create README.md

Fanis-Bazilisk committed:
Update Java code connect to Arduino

Fanis-Bazilisk committed:
test

Fanis-Bazilisk committed:
Arduino

Co-Authored-By: TimofeiV <72314768+TimofeiV@users.noreply.github.com>

Fanis-Bazilisk committed:
Setting Arduino from Java

Fanis-Bazilisk committed:
Connect and Read Arduino

Fanis-Bazilisk committed:
Merge pull request #2 from Fanis-Bazilisk/main

Main

Fanis-Bazilisk committed:
Update README.md

Fanis-Bazilisk committed:
Update README.md

Fanis-Bazilisk committed:
Update ArduinoReader

TimofeiV commented on ArduinoJAVA:
```diff
@@ -0,0 +1,29 @@
+//Подключение к ардуино через библиотеку jssc 2.9.2
+//Настройка
+package newtest;
```
Что делает ArduinoJAVA программа- просто проверяет что все законнектилось?

Fanis-Bazilisk replied:
> Что делает ArduinoJAVA программа- просто проверяет что все законнектилось?

Настраивает порт и проверяет на коннект ардуино.

Fanis-Bazilisk committed:
Добавил массив в который добавляю поток данных из ардуино и еще добавил поиск медианы в массиве по окну из определенных количеств чисел.

Поиск медианы не корректно срабатывает, также добавление в массив потока данных из ардуино некорректный.
